emission/core/wrapper/confirmedtrip.py

```diff
@@ -19,6 +19,7 @@ class Confirmedtrip(ecwt.Trip):
 # the user input will have all `manual/*` entries
 # let's make that be somewhat flexible instead of hardcoding into the data model
                   "user_input": ecwb.WrapperBase.Access.WORM
+                  "itinerum_boolean": ecwb.WrapperBase.Access.WORM
```

Comment:
To clarify, are we not touching the confirmed trip object at all? Can you elaborate on how the confirmed trip fits into the entire intake pipeline and what is not included within the confirmed trip?

Reply:
correct. The pipeline is structured so that each stage takes in an input and generates a distinct output. We don't modify inputs, consistent with standard pipeline guidelines. I document most of my design in issues - e.g. 
https://github.com/e-mission/e-mission-docs/issues/476